napari/utils/colormaps/colormaps.py

```diff
@@ -438,4 +422,35 @@ def ensure_colormap_tuple(colormap: ValidColormapArg) -> Tuple[str, Colormap]:
             'Must be a {str, tuple, dict, vispy.colormap.Colormap}'
         )
 
-    return name, AVAILABLE_COLORMAPS[name]
+    return AVAILABLE_COLORMAPS[name]
+
+
+def ensure_colormap_tuple(colormap: ValidColormapArg,) -> Tuple[str, Colormap]:
```

Comment:
will we need this anymore once we have a class that has a `name` attribute?

One thing I was also playing with the other day was mucking with the `__new__` method of the `Colormap` class, such that calling `Colormap(ValidColormapArg)` always does the right thing.  The goal there would be to be able to use `Property[ValidColormapArg, None, Colormap]` as easily as we use our `StringEnums`... thoughts?

Reply:
Nope, just for backwards compatibility right now, i've only converted `Image` but will probably drop this when I'd all done

Reply:
> mucking with the __new__ method of the Colormap class, such that calling Colormap(ValidColormapArg) always does the right thing. The goal there would be to be able to use Property[ValidColormapArg, None, Colormap] as easily as we use our StringEnums... thoughts

This sounds quite nice!

Reply:
But I will add I think it is also nice to have all the logic for going from string to a colormap outside the colormap object. I.e. `ensure_colormap` is actually quite intense, but `Colormap` right now is very simple. I like them being well separated.

Reply:
it could mostly remain that way probably... also, there will probably be some pushback in general to touching `__new__`, since I think people tend not to like that :joy: